report/3.Theory/LanguageSpecification.tex

```diff
@@ -49,21 +49,27 @@ \section{Language Specification}
 \textbf{Function Declaration}\\
 The type of a functions parameters, as well as its return type, are explicitly declared when a function is defined. 
 The declaration is structured as an assignment, where the functions name is declared on the left-hand-side, prefixed by either \textbf{\textit{var}} or \textbf{\textit{func}}. 
-The parameters of the function is declared on the right-hand-side inside parentheses, followed by an arrow (\textbf{\textit{->}}), which declares the return type. 
+The parameters of the function is declared on the right-hand-side inside parentheses, followed by a colon (\textbf{\textit{:}}), which declares the return type. 
 
-An example function that adds two numbers together and prints the value is shown below:
+An example function that adds two numbers together and returns the value is shown below:
 \begin{lstlisting}
-var addNumbers = (num a, num b) -> none {
-	a + b print
+var addNumbers = (num a, num b): num {
+    a + b
 }
 \end{lstlisting} 
 A more explicit declaration of the same function can be seen in the snippet below:
 \begin{lstlisting}
-func<num,num,none> addNumbers = (num a, num b) -> none {
-	a + b print
+func[num,num,none] addNumbers = (num a, num b): num {
+	return a + b
 }
 \end{lstlisting}
 
+As you may have noticed the return statement is missing in the first function. 
+"return" is optional if the function body only contains a single expression. 
+This makes sense when using lambdas, so you do not have to explicitly return the expression always. 
```

Comment:
dont talk directly to the reader
"..using lambdas, where return statements are often implicitly written." - or something like that